industrial_ci/src/docker.sh

```diff
@@ -111,7 +111,7 @@ function ici_run_cmd_in_docker() {
   wait %% || ret=$?
   trap - INT
   if [ -n "$commit_image" ]; then
-    docker commit -m "$_COMMIT_IMAGE_MSG" "$cid" $commit_image > /dev/null
```

Comment:
I'd like to keep `> /dev/null`, otherwise the SHA gets printed without context.
Instead a line could be printed: "Committing container to $DOCKER_COMMIT".